modules/SofaConstraint/GenericConstraintCorrection.h

```diff
@@ -49,6 +49,8 @@ class SOFA_CONSTRAINT_API GenericConstraintCorrection : public core::behavior::B
 
     void computeMotionCorrectionFromLambda(const core::ConstraintParams* cparams, core::MultiVecDerivId dx, const defaulttype::BaseVector * lambda) override;
 
+    void computeComplianceInConstraintSpace(const core::ConstraintParams *cparams, defaulttype::BaseMatrix* W) override;
+
     void addComplianceInConstraintSpace(const core::ConstraintParams *cparams, defaulttype::BaseMatrix* W) override;
```

Comment:
in fact before the #869 I think several methods were already virtual and nearly all method were virtual at the end of #869 but since that time I think some "clang clean PR" have removed many ```virtual``` keyword .
So when I rebased this this PR it was really a mess to know who should be virtual or not...